yesod-form/Yesod/Form/I18n/Chinese.hs

```diff
@@ -0,0 +1,26 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Yesod.Form.I18n.Chinese where
+
+import Yesod.Form.Types (FormMessage (..))
+import Data.Monoid (mappend)
+import Data.Text (Text)
+
+chineseFormMessage :: FormMessage -> Text
+chineseFormMessage (MsgInvalidInteger t) = "无效的整数: " `Data.Monoid.mappend` t
```

Comment:
for some reason, in all the language files (e.g., English.hs, Japanese.hs), the first line uses `Data.Monoid.mappend` while the other lines use just mappend.